specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -5092,6 +5092,10 @@
           "type": "boolean",
           "description": "Policy is enabled if set to true."
         },
+        "destination": {
+          "type": "string",
+          "description": "Deprecated Property from API version 2021-04-01 onwards, the required destination container name must be sepcified at the rule level 'poliy.rule.destination'"
```

Comment:
As the property is Deprecated , why we need add it?
We already has destination in each inventory policy rules, why add it here?

Reply:
We have S365 KPI incident which flags this. So we are adding it to get it right in s365 ratings

Reply:
If this is only for get inventory policy set on old API version, with SDK on new API version. We might should make this readonly, as it's only for get.

Reply:
Yes it is readOnly